Runtime/Scripts/glTFast.asmdef

```diff
@@ -25,17 +25,17 @@
             "define": "DRACO_UNITY"
         },
         {
-            "name": "net.vectorworks.ktx",
+            "name": "com.unity.cloud.ktx",
```

Comment:
There is a really unpleasant logic in the c# editor scripts that searches some .dll libs dynamically by the string `"com.unity.cloud.ktx"` and composes them during the build. I couldn't catch all the usages of the code and couldn't make it work, so I had to use the original name.